rustls/tests/common/mod.rs

```diff
@@ -284,7 +284,8 @@ pub fn get_client_root_store(kt: KeyType) -> RootCertStore {
 pub fn make_server_config_with_mandatory_client_auth(kt: KeyType) -> ServerConfig {
     let client_auth_roots = get_client_root_store(kt);
 
-    let client_auth = AllowAnyAuthenticatedClient::new(client_auth_roots);
+    // TODO(@cpu): allow building test configuration with CRLs.
+    let client_auth = AllowAnyAuthenticatedClient::new(client_auth_roots, Vec::default());
```

Comment:
I'll commit to writing some actual tests for this once we have the webpki side figured out and more of the questions in this branch are resolved.